ocs_ci/ocs/resources/pod.py

```diff
@@ -297,9 +310,25 @@ def run_git_clone(self):
         assert wl.setup(), "Setup up for git failed"
         wl.run()
 
+    def get_paths(self, pvcs_count, storage_type):
```

Comment:
I think it would be better if this function will just return all the paths of all the PVCs

Reply:
Do you mean get all the paths in the pod without taking pvc count?

Reply:
yes